src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/ProductSpecificPricePriorityType.php

```diff
@@ -0,0 +1,134 @@
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Pricing;
+
+use PrestaShop\PrestaShop\Adapter\Product\SpecificPrice\Repository\SpecificPriceRepository;
+use PrestaShop\PrestaShop\Core\Domain\Product\SpecificPrice\ValueObject\PriorityList;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class ProductSpecificPricePriorityType extends TranslatorAwareType
+{
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var SpecificPriceRepository
+     */
+    private $specificPriceRepository;
+
+    /**
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param RouterInterface $router
+     * @param SpecificPriceRepository $specificPriceRepository
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        RouterInterface $router,
+        SpecificPriceRepository $specificPriceRepository
+    ) {
+        parent::__construct($translator, $locales);
+        $this->router = $router;
+        $this->specificPriceRepository = $specificPriceRepository;
+    }
+
+    /**
+     * @param FormBuilderInterface $builder
+     * @param array<string, mixed> $options
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('use_custom_priority', ChoiceType::class, [
+                'choices' => [
+                    $this->buildDefaultPriorityChoiceLabel() => false,
+                    $this->trans('Set a specific order for this product', 'Admin.Catalog.Feature') => true,
+                ],
+                'default_empty_data' => false,
+                'placeholder' => false,
+                'expanded' => true,
+                'multiple' => false,
+                'required' => false,
+                'label' => false,
+                'external_link' => [
+                    'text' => $this->trans('[1]Manage default settings[/1]', 'Admin.Global'),
```

Comment:
```suggestion
                    'text' => $this->trans('[1]Manage default settings[/1]', 'Admin.Actions'),
```
Sorry I had missed this one